src/wfs/src/main/java/org/geoserver/wfs/response/ShapeZipOutputFormat.java

```diff
@@ -228,17 +228,14 @@ protected void shapefileDumped(
 
             // zip all the files produced
             final FilenameFilter filter =
-                    new FilenameFilter() {
-
-                        public boolean accept(File dir, String name) {
-                            name = name.toLowerCase();
-                            return name.endsWith(".shp")
-                                    || name.endsWith(".shx")
-                                    || name.endsWith(".dbf")
-                                    || name.endsWith(".prj")
-                                    || name.endsWith(".cst")
-                                    || name.endsWith(".txt");
-                        }
+                    (dir, name) -> {
+                        name = name.toLowerCase();
+                        return name.endsWith(".shp")
+                                || name.endsWith(".shx")
+                                || name.endsWith(".dbf")
+                                || name.endsWith(".prj")
+                                || name.endsWith(".cst")
+                                || name.endsWith(".txt");
```

Comment:
+1. Seems like we should have a defined list / array of all the sidecar files we produce though, instead of having it here.